airflow/www/views.py

```diff
@@ -2562,6 +2567,36 @@ class SlaMissModelView(AirflowModelView):
     }
 
 
+class ImportViewModelView(AirflowModelView):
```

Comment:
Can you post a screenshot of how this looks

Reply:
<img width="1438" alt="Screenshot 2020-11-20 at 15 08 36" src="https://user-images.githubusercontent.com/12058428/99809285-5ba04c80-2b42-11eb-8308-21642dfe50ca.png">
<img width="1432" alt="Screenshot 2020-11-20 at 15 09 26" src="https://user-images.githubusercontent.com/12058428/99809320-66f37800-2b42-11eb-93e8-42ccd114cb93.png">

Reply:
Nice looks good